sanic/request.py

```diff
@@ -296,6 +298,11 @@ def load_json(self, loads=json_loads):
 
         return self.parsed_json
 
+    @property
+    def accept(self) -> List[Accept]:
+        accept_header = self.headers.getone("accept", "")
+        return parse_accept(accept_header)
+
```

Comment:
Should this be cached? And could we have a decorator for caching instead of doing it manually like in most other `request` accessors? (does that affect performance?)

Reply:
That's a good idea.

Reply:
The reason I did not implement is that I did not want to waste resources adding extra values to the request object that would not be used in most scenarios. Maybe however the solution is to have a single `_cache` object that can contain these values.